tinygrad/helpers.py

```diff
@@ -79,31 +79,32 @@ def __init__(self, priority, itemsize, name, np, shape):
   def __repr__(self): return f"dtypes.{self.name}({self.shape})"
 
 class dtypes:
-  @staticmethod # static methds on top, or bool in the type info will refer to dtypes.bool
-  def is_int(x: DType)-> bool: return x in (dtypes.int8, dtypes.uint8, dtypes.int32, dtypes.int64)
-  @staticmethod
-  def is_float(x: DType) -> bool: return x in (dtypes.float16, dtypes.float32, dtypes.float64, dtypes._half4, dtypes._float4)
-  @staticmethod
-  def is_unsigned(x: DType) -> bool: return x in (dtypes.uint8, dtypes.uint32, dtypes.uint64)
   @staticmethod
   def from_np(x) -> DType: return DTYPES_DICT[np.dtype(x).name]
   @staticmethod
   def fields() -> Dict[str, DType]: return DTYPES_DICT
+  @staticmethod
+  def is_signed_int(x)-> bool: return x in (dtypes.int8, dtypes.int16, dtypes.int32, dtypes.int64)
+  @staticmethod
+  def is_unsigned_int(x) -> bool: return x in (dtypes.uint8, dtypes.uint16, dtypes.uint32, dtypes.uint64)
+  @staticmethod
+  def is_int(x) -> bool: return dtypes.is_signed_int(x) or dtypes.is_unsigned_int(x)
+  @staticmethod
+  def is_float(x) -> bool: return x in (dtypes.float16, dtypes.float32, dtypes.float64, dtypes._float4)
   bool: Final[DType] = DType(0, 1, "bool", bool)
-  float16: Final[DType] = DType(0, 2, "half", np.float16)
-  half = float16
```

Comment:
Why were these aliases removed?

Reply:
I removed float and half because I think its a bit confusing with two types which are identical. And I think float16 and float32 are more readable

Reply:
I do changes which I in the moment makes sense to me. Maybe for future PR I should discuss these thinks first